### PR TITLE
Kammy1996 patch 1

### PR DESCRIPTION
**Pull request by kammy1996**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4877"></a>

### CODE REVIEW

File: data/kammy1996.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Kamran Memon",
+  "type": "personal",
+  "bio": "I am a frontend Developer working with Vue.js",
+  "avatar" : "https://github.com/kammy1996.png",
```
Comment (priyankarpal) [inline data/kammy1996.json:5]: hey @kammy1996 please delete this `avatar` code. Currently, you don't need to add this command. 
``` suggestion
 
```

File: data/kammy1996.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "url": "https://twitter.com/kamranmemon25",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/kamran-memon-1003a2111/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Github"
```
Comment (priyankarpal) [inline data/kammy1996.json:18]: Add a comma `,`
``` suggestion
      "name": "Follow me on Github",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/kammy1996.json (hunk 1 above)
Comment (kammy1996) [reply data/kammy1996.json]: Changes done as per suggestion

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by Hrishi5111998 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**